packages/docs/src/pages/theming.mdx

```diff
@@ -181,6 +181,7 @@ Flag | Default | Description
 `initialColorModeName` | `'default'` | The key used for the top-level color palette in `theme.colors`
 `useColorSchemeMediaQuery` | `false` | Initializes the color mode based on the `prefers-color-scheme` media query
 `useBorderBox` | `true` | Adds a global `box-sizing: border-box` style
+`noLocalStorageForColorMode` | `false` | Disables the color mode being loaded from `localStorage`
```

Comment:
Thanks for looking into this. I think to remain grammatically consistent with the other options, this could be `useLocalStorage`, with a default to `true`

Reply:
Totally makes sense. Will update to match the other options' grammar.

Reply:
Alright, updated!